server/app/app.py

```diff
@@ -184,33 +188,38 @@ class SchemaAPI(DatasetResource):
     @cache_control(public=True, max_age=ONE_WEEK)
     @rest_get_data_adaptor
     def get(self, data_adaptor):
-        return common_rest.schema_get(data_adaptor, current_app.annotations)
+        annotations = data_adaptor.dataset_config.user_annotations
```

Comment:
given that `user_annotations` config is now managed by the data adaptor, why not push this code down into the rest.py layer, and not repeat it for each route handler?  (I may be missing the reason?)   Just seems like repetitive code in this layer, if annotations is always in the data_adaptor without fail.

Reply:
Yeah, that's a good idea.    No reason not to do it.